client/app/pages/alert/Alert.jsx

```diff
@@ -177,6 +178,17 @@ class Alert extends React.Component {
       });
   };
 
+  evaluate = () => {
+    const { alert } = this.state;
+    return AlertService.evaluate(alert)
+    .then(() => {
+      notification.success("Alert Evaluated. Please refresh the page.");
```

Comment:
Suggested wording: "Alert evaluated. Refresh page for updated status."

Reply:
I made these changes